internal/backend/db/dbgorm/gorm_test.go

```diff
@@ -21,10 +26,41 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-var now time.Time
+var (
+	now    time.Time
+	dbType string
+)
+
+func TestMain(m *testing.M) {
+	flag.StringVar(&dbType, "dbtype", "sqlite", "Database type to use for tests (e.g., sqlite, postgres)")
+	flag.Parse()
+
+	var exitCode int
+	if dbType == "postgres" {
+		pg := embedPG.NewDatabase() // embeddedpostgres.DefaultConfig.RuntimePath("/tmp"))
```

Comment:
this might belong in gormkitteh

Reply:
definitely. but there is a ticket to unify tests and gormkiteh.